modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -163,7 +152,7 @@ const ColorPicker = ({
   };
 
   return (
-    <Container data-testid="canvas-color-picker" {...elemProps}>
+    <Popup width={250} padding={Popup.Padding.s} data-testid="canvas-color-picker" {...elemProps}>
```

Comment:
I didn't notice the `data-testid`

Reply:
Yeah - I didn't worry about it too much because it was in the story and not baked into the component. But we can rework to remove this if you'd prefer.